pkg/motor/ssi.go

```diff
@@ -0,0 +1,21 @@
+package motor
+
+import (
+	"errors"
+)
+
+func (mtr *motorNodeImpl) AddBucketServiceEndpoint(id string) error {
```

Comment:
We'll need some way of committing these changes. Not sure if `updateWhoIs` needs to be done here or someplace else.